launchable/commands/record/case_event.py

```diff
@@ -51,14 +53,31 @@ def f(case: TestCase, suite: TestSuite, report_file: str) -> TestPath:
 
         return f
 
+    @staticmethod
+    def default_data_builder() -> DataBuilder:
+        def f(case: TestCase):
+            """
+            case for:
+                <testcase ... file="tests/commands/inspect/test_tests.py" line="133">
```

Comment:
Question: If there was a report that produces line number attribute as `line_number`, we add a line_number field to MetadataTestCase, right?

Reply:
Yes, but attributes for expressing line numbers, like `line_number`, are seldom seen. This is because the `line` attribute is usually used.

https://github.com/testmoapp/junitxml?tab=readme-ov-file#complete-junit-xml-example

Reply:
Got it